ENGAGEHF/HealthSummary/QRCodeShareView.swift

```diff
@@ -0,0 +1,62 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import CoreImage.CIFilterBuiltins
+import SwiftUI
+
+
+struct QRCodeShareView: View {
+    let url: String
+    let code: String
+    let context = CIContext()
+    let filter = CIFilter.qrCodeGenerator()
+    
+    
+    var body: some View {
+        VStack {
+            GroupBox(label: Text("Health Summary QR Code")) {
+                Text("This QR code can be scanned by your doctor to share your health summary.")
+                    .padding(.top)
+                Image(uiImage: generateQRCode(from: url))
+                    .interpolation(.none)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: 200, height: 200)
```

Comment:
Looks solid, I think we can even increase the QR code up to some horizontal padding.